pegasus/sites.v3/code.org/public/css/ai-pl-webinar-styles.scss

```diff
@@ -0,0 +1,108 @@
+// Custom styles for code.org/ai/pl/webinar
+// — supplemental to shared/css/design-system-pegasus.scss
+
+@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+
+.hero-ai-pl-webinar {
+  background-image: url(/images/ai/pl/webinar/hero--1.png), url(/images/ai/pl/webinar/hero--2.png);
+  background-position: left top, right bottom;
+  background-repeat: no-repeat, no-repeat;
+  background-color: #360161;
+  //background-size: contain;
+  background-size: 250px, 500px;
+  &__by {
+    display: flex;
+    align-items: center;
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    margin-left: 32px;
+    flex-wrap: wrap;
+
+    img {
+      margin: 16px 32px 16px 0;
+    }
+  }
+
+  .text-wrapper p {
+    font-size: 20px;
+    margin-bottom: 30px;
+  }
+
+  @media (max-width: 1499px) {
+    background-position: calc(20vw - 350px) top, right bottom;
+  }
+
+  @media (max-width: 1279px) {
+    background-image: url(/images/ai/pl/webinar/hero--2.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: 300px;
+    padding-bottom: 200px;
+  }
+
+  @media (max-width: 767px) {
+    &__by {
+      flex-direction: column;
+      align-items: flex-start;
+
+    }
+    &__list {
+      align-items: center;
+      margin-left: 0;
+      flex-wrap: wrap;
+
+      img {
+        margin: 12px 24px 12px 0;
+
+        &:last-child {
+          margin-right: 0;
+        }
+      }
+    }
+  }
+}
+
+.partnership-ai {
+  background: $neutral_light;
+
+  &__content {
```

Comment:
[Here are some flexbox (and grid) classes](https://github.com/code-dot-org/code-dot-org/blob/14759493513d5494781ee5d30e08d083d1b5845c/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L80-L102) you can use on these containers in the haml markup that negates the need to put them in the CSS.